lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1831,6 +1833,7 @@ export class UserAgentApplication {
           // retrieve the id_token from response if present :
           if (hashParams.hasOwnProperty(Constants.idToken)) {
             response.idToken = new IdToken(hashParams[Constants.idToken]);
+            response.claims = response.idToken.decodedIdToken;
```

Comment:
Why reassign a value that is already in the object? Seems like we are starting to have duplicate values in multiple places. This will start bloating objects, I think we should try to move away from this practice. If we need to restructure account and response to do this I am all for it.

Reply:
One option is to have it only in 'Account' but as stated in the above comment, it ties up in two different flows, request-response  (and the corresponding look up for claims); Account for generic session information. Let me get back on this after further evaluation.

Reply:
@pkanher617  I think what was talked about was getting rid of 'idToken' and so having 'claims' also available here lets us not break those looking at id token, and we can deprecate it, next major version.

@sameerag  if that is infact what we are doing, can we add a comment here explaining it ?

Reply:
also will need to update docs to reflect this.

Reply:
I think what we agreed upon was to remove the IdToken object type and change it to a string. Then adding the claims object makes sense to me. We should only use IdToken internally since we are not exporting it as a type.

Reply:
Yes. What @DarylThayil is stating is different, he is mentioning the parameter in 'Account' and not the 'Response'.

I added what we agreed as a comment at the end of this PR. Will alert you guys when the code changes are available.

Reply:
Code changes added.